src/ThundraWrapper.ts

```diff
@@ -152,9 +150,13 @@ class ThundraWrapper {
     }
 
     async executeAfteInvocationAndReport(afterInvocationData: any) {
+        afterInvocationData.error ? InvocationSupport.setErrorenous(true) : InvocationSupport.setErrorenous(false);
+
         await this.executeHook('after-invocation', afterInvocationData, true);
         this.resetTime();
         await this.reporter.sendReports();
+
+        InvocationSupport.setErrorenous(false);
```

Comment:
To make it more safe, can we also have `InvocationSupport.setErrorenous(false);` at the beginning of `invoke` before executing `before-invocation` hooks